src/components/Form.js

```diff
@@ -374,80 +383,82 @@ class Form extends Component {
                     onSubmit={this.onSubmit}
                     mutators={{ ...arrayMutators }}
                     keepDirtyOnReinitialize={false}
-                    subscription={{ values: false, submitFailed: true, submitting: true }}
+                    subscription={{ values: false, submitFailed: true }}
                     initialValues={this.state.initialValues}
                     noValidate
                 >
                     { (formProps) => {
-                        const { handleSubmit, form, submitting, submitFailed } = formProps;
```

Comment:
seems this `submitting` not need to remove - its before onSubmit called (validation etc)